src/screens/FeedScreen/Header.js

```diff
@@ -139,6 +139,7 @@ const _renderAnonimity = ({
                 karmaScore={karmaScore}
                 anonUserInfo={anonUserInfo}
                 version={version}
+                withKarma
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `withKarma` prop is a good change if it's used to conditionally render some part of the component. However, without knowing what this prop does, it's hard to provide a more detailed review. Please ensure that the parent component passes this prop and that it's properly handled in the child component.
```diff
+                withKarma
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->